service/kvmconfig/v6/resource/service/master_service.go

```diff
@@ -23,6 +23,10 @@ func newMasterService(customObject v1alpha1.KVMConfig) *apiv1.Service {
 			},
 			Annotations: map[string]string{
 				"giantswarm.io/prometheus-cluster": key.ClusterID(customObject),
+				"prometheus.io/path":               "/healthz",
+				"prometheus.io/port":               "30010",
```

Comment:
443 ???

Reply:
please ignore my comment. all good.

Reply:
we don't wanna scrape api here, that is already taken care off by `prometheus-config-controller`
We discussed this with @JosephSalisbury and it's still beneficial for us to monitor master endpoint ip like this.